respy/pre_processing/model_checking.py

```diff
@@ -35,6 +31,46 @@ def validate_options(o):
         )
 
 
+def validate_params(params, optim_paras):
```

Comment:
I dont get the gain of this function am I missing something or is this style guide stuff?

Reply:
Previously we sorted the shocks within respy. Unfortunately, if we optimize with estimagic, estimagic assumes that the shocks are already in the correct format for the constraint before params even reaches respy. Now, we throw an error if the shock matrix is not correctly sorted.